download.py

```diff
@@ -1,8 +1,9 @@
 import pandas
+import os
 
 
-IN_FILE_NAME = "gs://genomics-public-data/simons-genome-diversity-project/reports/Simons_Genome_Diversity_Project_sample_reference_results.csv"
-OUT_FILE_NAME = "results.parquet"
+IN_FILE_NAME = os.environ.get('IN_FILE_NAME')
```

Comment:
Let's avoid `get` for environment variables that _must_ be provided because that means the script will silently fail later on in its execution, with a potentially misleading error message.